packages/cojson/src/coValues/coMap.ts

```diff
@@ -150,7 +152,19 @@ export class RawCoMapView<
 
     for (const [key, entries] of changedEntries.entries()) {
       this.latest[key] = entries[entries.length - 1];
+      const latestChange = this.latest[key]!.change;
+      if (latestChange.op === "set") {
```

Comment:
I guess we're fine with always paying this cost of generating diffs when processing transactions, right?

The alternative would be to only start doing it if someone's subscribed to the diffs